errors/apiErrors.ts

```diff
@@ -116,16 +112,15 @@ export function parseValidationErrors(
  * @throws
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-function throwValidationError(error: AxiosError<any>) {
+function throwValidationError(error: HubSpotHttpError) {
   const validationErrorMessages = parseValidationErrors(error?.response?.data);
   if (validationErrorMessages.length) {
     return new Error(validationErrorMessages.join(' '), { cause: error });
   }
 }
 
-export function getAxiosErrorWithContext(
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  error: AxiosError<any>,
+export function getHubSpotHttpErrorWithContext(
```

Comment:
Just recording here that this will be a major version bump since it's a breaking change (totally fine, just wanted to add a comment for posterity)